src/python/tests/test_libs/test_utils.py

```diff
@@ -384,3 +387,14 @@ def decorator(func):  # pylint: disable=unused-argument
                                func)
 
   return decorator
+
+
+class MockStdout(io.BufferedWriter):
```

Comment:
this was needed because the new io.StringIO and io.BytesIO are now strict about str vs unicode